nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/SFTPTransfer.java

```diff
@@ -580,12 +587,23 @@ private String getMessage(final SFTPException e) {
     }
 
     protected SFTPClient getSFTPClient(final FlowFile flowFile) throws IOException {
+        final String evaledHostname = ctx.getProperty(HOSTNAME).evaluateAttributeExpressions(flowFile).getValue();
+        final String evaledPort = ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).getValue();
+        final String evaledUsername = ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).getValue();
+        final String evaledPassword = ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).getValue();
+        final String evaledPrivateKeyPath = ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).getValue();
+        final String evaledPrivateKeyPassphrase = ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).getValue();
```

Comment:
It looks like the property descriptor `PORT` reference needs to be replaced with the appropriate property descriptor for the username, password, and private key values.